node/runtime/src/lib.rs

```diff
@@ -59,7 +59,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	impl_name: create_runtime_str!("substrate-node"),
 	authoring_version: 10,
 	spec_version: 92,
-	impl_version: 93,
+	impl_version: 94,
```

Comment:
As we don't change anything in the runtime, we don't need to bump the `impl_version`.